tests/ctests/test_hybrid_merge.c

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2006-Present, Redis Ltd.
+ * All rights reserved.
+ *
+ * Licensed under your choice of the Redis Source Available License 2.0
+ * (RSALv2); or (b) the Server Side Public License v1 (SSPLv1); or (c) the
+ * GNU Affero General Public License v3 (AGPLv3).
+*/
+#include "hybrid/hybrid_search_result.h"
+#include "rmutil/alloc.h"
+#include "test_util.h"
+#include "redisearch.h"
+#include "rlookup.h"
+#include "value.h"
+#include <math.h>
+#include <string.h>
+#include <stdio.h>
+#include <assert.h>
+
```

Comment:
This file name is a bit weird w.r.t to the tests that are actually here.
Consider embedding these tests in [tests/cpptests/test_cpp_hybridmerger.cpp](https://github.com/RediSearch/RediSearch/pull/6579/files?show-viewed-files=true&file-filters%5B%5D=#diff-170964ee10f784ac04ffc49d257166ad091dbd320465b052d839223931f0bc4e)